### PR TITLE
Update SqlFacetAttribute.xml to indicate possible usage on input parameters

### PR DESCRIPTION
**Pull request by allmhuran**
Document implies SqlFacets can only be used with return types. But they can also be used with input parameters.

## Summary

Describe your changes here.

Fixes #Issue_Number (if available)

### CODE REVIEW

File: xml/Microsoft.SqlServer.Server/SqlFacetAttribute.xml [hunk 1]
```diff
@@ -32,7 +32,9 @@
       <format type="text/markdown"><![CDATA[  
   
 ## Remarks  
- <xref:Microsoft.SqlServer.Server.SqlFacetAttribute> may only be specified on non-void return values.  
+ <xref:Microsoft.SqlServer.Server.SqlFacetAttribute> may only be specified on non-void return values, or on input parameters.
```
Comment (gewarren) [inline xml/Microsoft.SqlServer.Server/SqlFacetAttribute.xml:35]:
```suggestion
 <xref:Microsoft.SqlServer.Server.SqlFacetAttribute> can only be specified on non-void return values or on input parameters. 
```

File: xml/Microsoft.SqlServer.Server/SqlFacetAttribute.xml [hunk 2]
```diff
@@ -32,7 +32,9 @@
       <format type="text/markdown"><![CDATA[  
   
 ## Remarks  
- <xref:Microsoft.SqlServer.Server.SqlFacetAttribute> may only be specified on non-void return values.  
+ <xref:Microsoft.SqlServer.Server.SqlFacetAttribute> may only be specified on non-void return values, or on input parameters. 
+
+ to specify the <xref:Microsoft.SqlServer.Server.SqlFacetAttribute> on an input parameter, add the decoration inline with the parameter.
```
Comment (gewarren) [inline xml/Microsoft.SqlServer.Server/SqlFacetAttribute.xml:37]:
```suggestion
To specify the <xref:Microsoft.SqlServer.Server.SqlFacetAttribute> on an input parameter, add the decoration inline with the parameter.
```

**Comment by gewarren** [comment]
@stevestein Can you review this please?

**Comment by allmhuran** [comment]
@stevestein When publishing  a SQLCLR UDF via Visual Studio, the user in the stackoverflow thread (below) found that the deployed object treated the input parameter numeric as (18,0). The attribute was able to be used to inform the deployment. The user was given the impression that the facet could only be used on return, as I believe is implied by the existing documentation. Solomon Rutzky indicated that, regarding input or output, the facet "is definitely intended for both, even if the examples are lacking".

Perhaps there is a more comprehensive way to indicate this in the documentation than my proposed edit.

https://stackoverflow.com/questions/62773147/set-decimal-precision-in-sql-clr-function-in-c-sharp

**Comment by carlossanlop** [comment]
Ping @stevestein - can you please follow up?

**Comment by BillWagner** [comment]
ping @carlossanlop @stevestein to follow up.

**Review by DavoudEshtehari — APPROVED** [review]

**Comment by allmhuran** [comment]
@dotnet-policy-service agree

**Review by gewarren — APPROVED** [review]

**Pull request merged**